website/translated_docs/ar/learn-consensus.md

```diff
@@ -6,41 +6,51 @@ sidebar_label: Polkadot Consensus
 
 ## Why do we need consensus?
 
-Consensus is a method for coming to agreement over a shared state. In order for the state of the blockchain to continue to build and move forward, all nodes in the network must agree and come to consensus. It is the way that the nodes in a decentralized network are able to stay synced with each other. Without consensus for the decentralized network of nodes in a blockchain, there is no way to ensure that the state one node believes is true will be shared by the other nodes. Consensus aims to provide the *objective* view of the state amid participants which each have their own *subjective* views of the network. It is the process for which these nodes communicate and come to agreement, and are able to build new blocks.
+Consensus is a method for coming to agreement over a shared state. In order for the state of the blockchain to continue to build and move forward, all nodes in the network must agree and come to consensus. It is the way that the nodes in a decentralized network are able to stay synced with each other. Without consensus for the decentralized network of nodes in a blockchain, there is no way to ensure that the state one node believes is true will be shared by the other nodes. Consensus aims to provide the _objective_ view of the state amid participants which each have their own _subjective_ views of the network. It is the process by which these nodes communicate and come to agreement, and are able to build new blocks.
```

Comment:
This is outside the scope of this PR, but why are there all this English text in the translations?

@ansonla3 Can we remove these placeholders or will they cause issues with crowdin?

Reply:
Yeah it was a little confusing but all the translations were English. So I modified these files as well. Would love clarification on how this actually works. If the translation files are somehow autogenerated, we can just delete the translated docs folder maybe?

Reply:
We need the English files so that it displays the English version of the page if we do not have the translated content. CrowdIn will handle creating these and keeping these sync automatically. As an author in English, you only need to worry about editing the source files. CrowdIn will routinely create a pull request that will bring in the updated pages.

Reply:
Okay, so good to skip the translations, nice. Thanks!